devdoc/userdoc/JiraGuidelines.md

```diff
@@ -3,13 +3,13 @@
 This is a guide to the type of information that is desired in JIRA tickets.
 
 # For bugs / improvement requests on any existing features or docs: 
-   - Provide a screenshot when it’s applicable.  Not just of the offending line or excerpt, but of the entire application window.
-      - This is especially true for issues related to ECL Watch.
+   - User-Interface Issues:
+      - Provide a screenshot when it’s applicable.  Not just of the offending line or excerpt, but of the entire application window.
+      - URL to environment.
```

Comment:
url to environment and workunit are relevant to all issues.

Which browser and version is only relevant to UI.